cloudant-sync-datastore-core/src/test/java/com/cloudant/sync/internal/query/IndexUpdaterTest.java

```diff
@@ -205,7 +205,7 @@ public void run() {
                     rev.setBody(DocumentBodyFactory.create(bodyMap));
                     try {
                         ds.createDocumentFromRevision(rev);
-                    } catch (DocumentException de) {
+                    } catch (Exception de) {
```

Comment:
Although it doesn't really matter, it might be worth changing variable `de` to `e` just to save anyone wondering in what `de` means in future.